sirepo/resource.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+u"""Functions for accessing resource files
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from pykern import pkio
+from pykern import pkresource
+import os
+import sirepo.feature_config
+import werkzeug.utils
+
+
+def filename(*paths):
+    """Get absolute path to a resource file
+
+    Args:
+        paths (str): Names of paths to get to resource file
+
+    Returns:
+        py.path: absolute path to resource file
+    """
+    return pkio.py_path(pkresource.filename(
+        os.path.join(*paths),
+        additional_packages=sirepo.feature_config.dynamic_sim_type_packages(),
+    ))
+
+
+def glob_static(path):
```

Comment:
static_paths_for_type might be a better name (see comment elsewhere about specifying interface).

Reply:
eb1b916d